Modules/IO/GE/src/itkGE4ImageIO.cxx

```diff
@@ -344,11 +345,11 @@ GE4ImageIO::ReadHeader(const char * FileNameToRead)
 float
 GE4ImageIO ::MvtSunf(int numb)
 {
-#define signbit 020000000000U
-#define dmantissa 077777777U
-#define dexponent 0177U
-#define smantissa 037777777U
-#define smantlen 23U
+  constexpr auto signbit = 020000000000U;
+  constexpr auto dmantissa = 077777777U;
+  constexpr auto dexponent = 0177U;
+  constexpr auto smantissa = 037777777U;
```

Comment:
I would rather see my 4-byte bit masks described in hexadecimal than in octal:
```cpp
  constexpr auto signbit   = 0x80000000U;
  constexpr auto dmantissa = 0x00FFFFFFU;
  constexpr auto dexponent = 0x7FU;
  constexpr auto smantissa = 0x007FFFFFU;
```

Also, does the trailing `U` count as a use of the type on the right-hand side, so that `auto` is appropriate on the left-hand side?  (Regardless, it is definitely a step up from the `#define`.)

Reply:
> Personally, I would rather see my 4-byte bit masks described in hexadecimal than in octal

Thanks for the suggestions. Hexadecimal looks more common to me as well, but as it wasn't originally my code, I'd rather leave it octal for now.

>  Also, does the trailing `U` count as a use of the type on the right-hand side, so that auto is appropriate on the left-hand side?

I think so, yes.